common/persistence/visibility/store/cassandra/query_filters.go

```diff
@@ -0,0 +1,142 @@
+package cassandra
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/xwb1989/sqlparser"
+	"go.temporal.io/api/enums/v1"
+)
+
+type QueryFilters struct {
+	HasWorkflowIDFilter bool
+	WorkflowID string
+	HasWorkflowTypeFilter bool
+	WorkflowType string
+	HasExecutionStatusFilter bool
+	ExecutionStatus enums.WorkflowExecutionStatus
+	HasStartTimeRangeFilter bool
+	StartTimeFrom time.Time
+	StartTimeTo time.Time
```

Comment:
Do we require to have both of from and to? I think it is valid to query with only from or to? @alexshtin

Reply:
There is validation on FE handler side.